easybuild/easyconfigs/s/sratoolkit/sratoolkit-2.5.2.eb

```diff
@@ -0,0 +1,35 @@
+import time
+
+easyblock = 'PackedBinary'
+
+name = 'sratoolkit'
+version = '2.5.2'
+
+homepage = 'http://www.ncbi.nlm.nih.gov/Traces/sra/sra.cgi?view=software'
+description = """The SRA Toolkit, and the source-code SRA System Development Kit (SDK), will allow you to programmatically access data housed within SRA and convert it from the SRA format"""
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+source_urls = ['http://ftp-trace.ncbi.nlm.nih.gov/sra/sdk/%(version)s/']
+sources = ['sratoolkit.%(version)s-ubuntu64.tar.gz']
+
+executables = ['abi-dump','blastn_vdb','fastq-load','kar','pacbio-load.2.5.2','sam-dump.2.5.2','srapath.2.5.2','srf-load.2.5.2','vdb-copy.2.5.2','vdb-lock.2.5.2',
```

Comment:
this blurb will need some reformatting...

these should be sorted alphabetically, for one

also, only the names should be specified, the `.2` and `.2.5.2` versions should generated